diff --git a/node_modules/@near-wallet-selector/near-wallet/index.cjs.js b/node_modules/@near-wallet-selector/near-wallet/index.cjs.js
index df3d149..a1327ca 100644
--- a/node_modules/@near-wallet-selector/near-wallet/index.cjs.js
+++ b/node_modules/@near-wallet-selector/near-wallet/index.cjs.js
@@ -1092,7 +1092,7 @@ function setupNearWallet({
         name: "NEAR Wallet",
         description: null,
         iconUrl,
-        deprecated: true,
+        deprecated: false,
         available: true
       })
     });
diff --git a/node_modules/@near-wallet-selector/near-wallet/index.esm.js b/node_modules/@near-wallet-selector/near-wallet/index.esm.js
index 6ac44d6..9651cec 100644
--- a/node_modules/@near-wallet-selector/near-wallet/index.esm.js
+++ b/node_modules/@near-wallet-selector/near-wallet/index.esm.js
@@ -1088,7 +1088,7 @@ function setupNearWallet({
         name: "NEAR Wallet",
         description: null,
         iconUrl,
-        deprecated: true,
+        deprecated: false,
         available: true
       })
     });
diff --git a/node_modules/@near-wallet-selector/near-wallet/index.umd.js b/node_modules/@near-wallet-selector/near-wallet/index.umd.js
index 432a862..5e47b6d 100644
--- a/node_modules/@near-wallet-selector/near-wallet/index.umd.js
+++ b/node_modules/@near-wallet-selector/near-wallet/index.umd.js
@@ -1092,7 +1092,7 @@
 	        name: "NEAR Wallet",
 	        description: null,
 	        iconUrl,
-	        deprecated: true,
+	        deprecated: false,
 	        available: true
 	      })
 	    });
